setup.py

```diff
@@ -3,7 +3,7 @@
 
 from setuptools import setup
 
-with open("README.md") as f:
+with open("README.md", encoding="utf8") as f:
     long_description = f.read()  # pylint: disable=invalid-name
```

Comment:
Not currently throwing a warning, but there have been cases where pylint considers `f` an invalid variable name. Our `self_check` level in Statick allows it as a good name, but this will make it work with vanilla pylint too.

https://github.com/sscpac/statick/blob/main/statick_tool/rsc/config.yaml#L142

```suggestion
with open("README.md", encoding="utf8") as fid:
    long_description = fid.read()  # pylint: disable=invalid-name
```